docs/developer_usage.rst

```diff
@@ -37,49 +37,51 @@ Checklist
 
-* This ``click`` group will contain all the subcommands under the ``<subtool>``'s functionality
+* This ``click`` group will contain all the functionality under the ``<tool>``
 
-5. Create separate files as needed for different subcommands; do not code out the full
-   implemetation of ``<subtool>`` inside of a ``click`` command within ``fre/<subtool>/fre<subtool>.py``.
+5. Create separate files as needed for different commands; do not code out the full
+   implemetation of ``<tool>`` inside of a ``click`` command within ``fre/<tool>/fre<tool>.py``.
 
-* better yet, consider what structure your subtool may need in the future for maintainability's sake
+* better yet, consider what structure your tool may need in the future for maintainability's sake
+* if you need, specify a ``<subtool>`` like ``fre/<tool>/<subtool>``. ``fre/app`` currently has
+  this structure
 
-6. Be sure to import the contents of the needed subcommand scripts inside of ``fre<subtool>.py``
+6. Be sure to import the contents of the needed subcommand scripts inside of ``fre<tool>.py``
 
-* i.e. from ``fre.fre<subtool>.subCommandScript import *``
+* i.e. from ``fre.fre<tool>.toolCommandScript import *``
 
-7. At this point, you can copy and paste the parts of your main ``click`` subcommand from its script
-   into ``fre<subtool>.py`` when implementing the function reflective of the subcommand function
+7. At this point, you can copy and paste the parts of your main ``click`` command from its script
+   into ``fre<tool>.py`` when implementing the function reflective of the command function
 
 * Everything will remain the same; i.e. arguments, options, etc.
 
-* However, this new function within ``fre<subtool>.py`` must a new line after the arguments, options,
+* However, this new function within ``fre<tool>.py`` must a new line after the arguments, options,
   and other command components; ``@click.pass_context``
 
 * Along with this, a new argument ``context`` must now be added to the parameters of the command
   (preferably at the beginning, but it won't break it if it's not)
 
 8. From here, all that needs to be added after defining the command with a name is
-   ``context.forward(mainFunctionOfSubcommand)``, and done!
+   ``context.forward(mainFunctionOfToolCommand)``, and done!
 
-9. After this step, it is important to add ``from fre.fre<subtool> import`` to ``__init__.py``
+9. After this step, it is important to add ``from fre.fre<tool> import`` to ``__init__.py``
    within the /fre folder
 
-10. The last step is to replicate the subcommand in the same way as done in ``fre<subtool>.py``
-	inside of ``fre.py``, but make sure to add ``from fre import fre<subtool>`` and
-	``from fre.fre<subtool>.fre<subtool> import *``
+10. The last step is to replicate the command in the same way as done in ``fre<tool>.py``
+	inside of ``fre.py``, but make sure to add ``from fre import fre<tool>`` and
+	``from fre.fre<tool>.fre<tool> import *``
```

Comment:
I also think we took these out. Now it's just Lazy loading click groups I believe

Reply:
we're gonna go over this with a fine tooth comb more than a few times too!

big-swings largely at structure, this PR is. detail oriented, it is not completely